dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/JdbiRepositoryImpl.java

```diff
@@ -258,14 +258,50 @@ protected String getWhereClause(
           break;
         case CONTAINS:
           // @see https://www.postgresql.org/docs/11/functions-matching.html
-          query
-              .append("(")
-              .append(expression)
-              .append(" ILIKE '%' || ")
-              .append(":")
-              .append(criterionKey)
-              .append(" || '%')");
-          argumentMappings.put(criterionKey, fc.getValue());
+
+          // Collections: siehe SearchTermTemplates " DB @> query"
+          if (fc.getValue() instanceof Collection<?> valueCollection
+              && !((Collection<?>) fc.getValue()).isEmpty()) {
```

Comment:
sollten wir hier schon nehmen können
```suggestion
              && !valueCollection.isEmpty()) {
```

Reply:
fixed